src/PostgresEventStore.php

```diff
@@ -478,7 +478,10 @@ private function addStreamToStreamsTable(Stream $stream): void
         ]);
 
         if (! $result) {
-            throw StreamExistsAlready::with($stream->streamName());
+            if (in_array($statement->errorCode(), $this->persistenceStrategy->uniqueViolationErrorCodes())) {
+                throw StreamExistsAlready::with($stream->streamName());
+            }
+            throw new RuntimeException('Unknown error. Maybe the event streams table is not setup?');
```

Comment:
$statement->errorInfo() provides an array with error messages. Maybe you add this strings to get a more meaningful error message?